scripts/inline_comments_data_collection.py

```diff
@@ -136,18 +107,18 @@ def process_comments(limit, diff_length_limit):
             if not most_recent_update:
                 continue
 
-            fix_patch_id, revision_phid = get_diff_info_from_phid(
-                most_recent_update["fields"]["new"]
-            )
+            fix_patch_id = diff_phid_to_id.get(most_recent_update["fields"].get("new"))
 
-            # If the most recent patch is the original patch itself, skip it
-            if fix_patch_id == patch_id:
+            # If the  most recent patch doesn't exist or is the original patch itself, skip it
+            if not fix_patch_id or fix_patch_id == patch_id:
                 continue
 
-            revision_id, bug_id = find_details_from_revision_phid(phid=revision_phid)
+            revision_phid = revision_info.get("phid")
+            revision_id = revision_info.get("id")
```

Comment:
For the keys that are always available, don't use `get`. This will make it easier to prevent errors in the future (e.g. if the key changes name, we won't start silently failing but we will see an exception).

Reply:
Makes sense, replaced `.get()` with direct accesses: 931333c920ee853a98be1e0a2f7bd9d193a31d37